src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -60,19 +60,26 @@ export const PaymentContent: FC<Props> = props => {
   } = props
   const tracking = useTracking()
 
-  const trackClickedPaymentMethod = (val: string): void => {
-    const event = {
-      subject: "click_payment_method",
-      payment_method: val,
-      action: ActionType.clickedPaymentMethod,
-      flow: order.mode!,
-      context_page_owner_type: OwnerType.ordersPayment,
-      order_id: order.internalID,
-      amount: order.buyerTotal,
-      currency: order.currencyCode,
+  const previousPaymentMethod = useRef<string | null>(null)
+
+  useEffect(() => {
+    if (paymentMethod !== previousPaymentMethod.current) {
```

Comment:
**thought (non-blocking):** `previousPaymentMethod.current` is a bit of a mind bender 😄